idea-plugin/src/main/java/net/jangaroo/ide/idea/JangarooFacetImporter.java

```diff
@@ -470,10 +472,26 @@ private void configureMxmlNamespaces(MavenProject mavenProjectModel, ModifiableF
         }
         String namespaceMapping = formatNamespaceMapping(namespacesToManifests);
         if (namespaceMapping != null) {
-          Map<String, String> allOptions = Collections.singletonMap("compiler.namespaces.namespace", namespaceMapping);
-          buildConfiguration.getCompilerOptions().setAllOptions(allOptions);
+          allOptions.put("compiler.namespaces.namespace", namespaceMapping);
         }
       }
+      Element extNamespaceElement = jangarooPluginConfiguration.getChild("extNamespace");
+      if (extNamespaceElement != null) {
+        String extNamespace = extNamespaceElement.getTextTrim();
+        if (!".".equals(extNamespace)) {
+          allOptions.put("extNamespace", extNamespace);
+        }
+      }
+      Element extSassNamespaceElement = jangarooPluginConfiguration.getChild("extNamespace");
+      if (extSassNamespaceElement != null) {
+        String extSassNamespace = extSassNamespaceElement.getTextTrim();
+        if (!".".equals(extSassNamespace)) {
+          allOptions.put("extSassNamespace", extSassNamespace);
+        }
+      }
+      if (!allOptions.isEmpty()) {
```

Comment:
This method also updates an existing buildConfiguration. This means it could happen (while unlikely) that the buildConfiguration _becomes_ empty, e.g. when you remove all corresponding elements from the Maven <configuration>. So I think you should _always_ setAllOptions(), even when allOptions is empty.

Reply:
So we are fixing a potentional bug here? The current behavior is the same as before.